static/app.css

```
@@ -7,39 +7,54 @@ html, body {
 body {
 	flex: 1 0 100%;
 	margin: 0;
-	background: #eee;
 	font-size: 1.25rem;
-	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif;
+	font-family: 'Kanit', sans-serif;
```

Would it make sense to keep the system fonts as fallback?
```suggestion
	font-family: 'Kanit', -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif;
```

---

sounds good